doc/2/guides/main-concepts/5-authentication/index.md

```diff
@@ -2,9 +2,362 @@
+
+**Example:** _Create an user with `local` credentials_
+```bash
+kourou security:createUser '{
+  content: {
+    profileIds: ["default"]
+  },
+  credentials: {
+    local: {
+      username: "mylehuong",
+      password: "password"
+    }
+  }
+}'
+```
+
+They will then be **stored by the plugin** in charge of the `local` strategy in a **secure storage space** accessible only with the code of this plugin.
+
+It is possible to manipulate a user's credentials:
+ - [security:getCredentials](/core/2/api/controllers/security/get-credentials): retrieve credentials information for a strategy
+ - [security:createCredentials](/core/2/api/controllers/security/create-credentials): create new credentials another strategy
+ - [security:deleteCredentials](/core/2/api/controllers/security/delete-credentials): delete credentials for a strategy
+
+When a user wants to authenticate to Kuzzle, he must choose a strategy and then provide the information requested by the strategy.
+
+For example for the `local` strategy it is required to provide a `username` and a `password`:
+
+```bash
+kourou auth:login -a strategy=local --body '{
+  username: "mylehuong",
+  password: "password"
+}'
+```
+
+## Authentication Token
+
+Authentication is performed using the [auth:login](/core/2/api/controllers/auth/login) API action.  
+
+This action requires the name of the strategy to be used as well as any information necessary for this strategy.
+
+When authentication is successful, **Kuzzle returns an authentication token**. This token has a validity of 2 hours by default, then it will be necessary to refresh it or to ask for a new one.
+
+::: info
+It is possible to request a token authentication valid for more than 2 hours with the argument `expiresIn`.  
+The default validity period is configurable under the key `security.jwt.expiresIn`.  
+It is also possible to set a maximum validity period for a token under the key `security.jwt.maxTTL`.
+:::
+
+This token must then be [provided in requests to Kuzzle API](/core/2/some-link) to authenticate the user.
+
+::: warning
+For historical reasons the API terminology uses the term `jwt` but Kuzzle authentication tokens only have in common with JSON Web Tokens the algorithms used to generate and verify them.
+:::
+
+Authentication tokens are revocable using the [auth:logout](/core/2/api/controllers/auth/logout) API action.
+
+### Authentication Token Expiration
+
+Authentication **token expires after a defined period of time**. Once an authentication token has expired, it **cannot be used in any way**.  
+
+::: info
+If the customer had subscribed to real-time notifications then they will be notified at the time of expiration with a [TokenExpired server event](/core/2/some-link).
+:::
+
+It is possible to use the [auth:refreshToken](/core/2/api/controllers/auth/refresh-token) API action to increase the duration of a still valid token authentication.
```

Comment:
Technically, we don't increase the validity period of an existing token. Rather, this action allows to request a new token in lieu of the old one.

```suggestion
While an authentication token is still valid, it is possible to provide it to the [auth:refreshToken](/core/2/api/controllers/auth/refresh-token) API action to request a new, fresher authentication token, without having to ask for credentials. 
```